shared/cfg/guest-hw.cfg

```diff
@@ -7,6 +7,7 @@ variants:
         nic_model = rtl8139
     - e1000:
         no ppc64 ppc64le
+        no RHEL.8
```

Comment:
How about `only RHEL.6 RHEL.7`, thus we don't need to add RHEL.9 or later in the further.

Reply:
Good catch, but it should be:
```
RHEL:
    only RHEL.6 RHEL.7
```

Reply:
Updated. Please help review again. Thanks.